### PR TITLE
Updated Travis config after zonemaster-ldns#51

### PR DESCRIPTION
**Pull request by mattias-p**
zonemaster/zonemaster-ldns#51 adds feature checks for Ed25519, but Travis provides an older version of OpenSSL.

This is blocked by #540.

I'm disabling Perl 5.16 in Travis because the tests fail for this version. I've created #551 to re-enable it. @matsduf and I discussed this and it is our view that for this release it's better to disable these tests and have zonemaster/zonemaster-ldns#51 implemented, than not having that feature implemented but having enabled tests for Perl 5.16 in Zonemaster Backend.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,14 +20,17 @@ perl:
 before_install:
     - sudo apt-get install -y libidn11-dev
     - eval $(curl https://travis-perl.github.io/init) --auto
-    - local-lib
+
+    - sudo apt-get install -y libidn11-dev
```
Comment (matsduf) [inline .travis.yml]: Should that really be installed a second time?

Comment (mattias-p) [reply .travis.yml]: They should not. I'll remove one of the commands.

File: .travis.yml [hunk 2]
```diff
@@ -15,19 +15,85 @@ perl:
     - "5.24"
     - "5.22"
     - "5.20"
-    - "5.16"
```
Comment (matsduf) [inline .travis.yml]: I suggest that 5.16 is kept, but commented with a short notice of why.

**Comment by mattias-p** [comment]
@matsduf I created #551 to re-enable Perl 5.16, added an explanation regarding this to the PR description above, and a comment to the configuration about 5.16 disabled.

**Comment by mattias-p** [comment]
@sandoche2k, @mtoma Do you agree with disabling Perl 5.16 in Travis for the benefit of merging zonemaster/zonemaster-ldns#85?

**Pull request merged**